YunoHost/yunohost
# Enh system log

## The problem

Some system log are not available in web admin

## Solution

This PR add this log in Tools > Log like it was designed during the Log PR.
![system_logs](https://user-images.githubusercontent.com/4080016/63117321-d8399800-bf9b-11e9-9112-8e50a15d4490.png)



## PR Status

Not ready
This PR depends of an other on yunohost-admin

## How to test

We need to merge this in unstable to test completely, but you can test like this:

- Pull branch on yunohost and yunohost-admin
- use git on yunohost and yunohost-admin
- cp -R yunohost/data/other/logs/* /var/log/yunohost/categories/
- service yunohost-admin
- Try to use the webadmin to see the new log in screenshot

## Validation
